ote/src/clj/ote/integration/export/geojson.clj

```diff
@@ -65,21 +65,14 @@
 
 (defn- append-nap-generated-netex-file-links
   "Returns `service` collection where NAP NeTEx file download link is appended to interfaces collection."
-  [service db config transport-service-id]
+  [service db config]
   (if (feature/feature-enabled? config :netex-conversion-automated)
     (when service
-      (let [netex-conversions (fetch-conversions db transport-service-id)]
-        (update service
-                ::t-service/external-interfaces
-                #(vec
-                   (concat []
-                           %
-                           (for [conversion netex-conversions]
-                             {:format "NeTEx"
-                              :data-content (:ote.db.netex/data-content conversion)
-                              ::t-service/external-interface (export-netex/file-download-url config
-                                                                                             transport-service-id
-                                                                                             (:ote.db.netex/id conversion))}))))))
+      (-> (vector service)
+          (netex-util/append-ote-netex-urls config
+                                            db
+                                            ::t-service/external-interfaces)
+          first))
```

Comment:
```suggestion
          (first
            (netex-util/append-ote-netex-urls (vector service) config db ::t-service/external-interfaces))
```

Reply:
While this may be opinionated, PR's non-nested approach is more modular (maintainable). 
Referring e.g. to https://stuartsierra.com/2018/07/06/threading-with-style